packages/config/src/bridges/stargate.ts

```diff
@@ -20,7 +20,7 @@ export const stargate: Bridge = {
         'https://github.com/LayerZero-Labs/LayerZero',
       ],
       socialMedia: [
-        'https://discord.gg/agEXvDB',
```

Comment:
We required the discord link to be in the format `https://discord.gg/<code>` or `https://discord.com/invite/<code>`. The `https://stargate.finance/discord` is not going to work.

Reply:
@mateuszradomski I went with the first option (keep the old outdated link) and now the test passes. The other 3 updated links are working without problem.